pkg/feature/api/feature_test.go

```diff
@@ -1013,6 +1013,244 @@ func TestEvaluateFeatures(t *testing.T) {
 	}
 }
 
+func TestEvaluateSingleFeature(t *testing.T) {
```

Comment:
Can you add the test case if req.FeatureId is not set.

Reply:
@ono-max 
That case is already being tested in the `TestEvaluateFeatures`.
I split the test because it was too long.